govcd/entity_test.go

```diff
@@ -78,7 +78,7 @@ func (vm *VM) id() string   { return vm.VM.ID }
 // Get invalid name or ID
 // To use this function, the entity must satisfy the interface genericEntity
 // and within the caller it must define the getter functions
-//
+//AddToCleanupList(diskCreateParamsDisk.Name
```

Comment:
This is not a requirement to run these tests. Please remove.

Reply:
removed